tests/image-stream/tester.js

```diff
@@ -43,6 +43,8 @@ const deleteIfExists = (file) => {
   });
 };
 
+exports.DEFAULT_TIMEOUT = 20000;
```

Comment:
I think this should have a name something more like `DEFAULT_IMAGE_TESTS_TIMEOUT` ?